bases/world_wb_wwbi/country_finance/table_config.yaml

```diff
@@ -0,0 +1,469 @@
+      is_partition: false
+    - name: trade_system
+      bigquery_type: string
+      description: Type of trade system
+      temporal_coverage:
+          - 2022
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: balance_payments_manual
+      bigquery_type: string
+      description: Balance of Payments Manual in use
+      temporal_coverage:
+          - 2022
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: external_debt_reporting
+      bigquery_type: string
+      description: External debt Reporting status
+      temporal_coverage:
+          - 2022
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: government_accounting
+      bigquery_type: string
+      description: Government Accounting concept
+      temporal_coverage:
+          - 2022
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: population_census
+      bigquery_type: string
+      description: Latest population census
+      temporal_coverage:
+          - 2022
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: agricultural_census
+      bigquery_type: string
+      description: Latest agricultural census
+      temporal_coverage:
+          - 2022
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: household_survey
+      bigquery_type: string
+      description: Latest household survey
+      temporal_coverage:
+          - 2022
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: industrial_date
+      bigquery_type: int64
+      description: Latest industrial data
```

Comment:
é data ou date?

Reply:
eu atualizei a descrição e acabei não mexendo no nome da variável, pq na descrição antes tava "Date of the last data obtained", pq meio que a resposta é o nome dos últimos dados com a data, ai meio que aceita os dois... mas talvez tenha ficado confuso manter o `_date`, na descrição, vou att ela

Reply:
acho que vale colocar na descrição que a variavel se refere ao ano. 'Year of the Latest...'

Reply:
alterei e acrescentei nas observações a descrição de como alguns dados vão aparecer, pra explicar melhor o problema `nome pesquisa + ano`